File: Tarea8/Usuario.java
import java.util.ArrayList;

import java.util.Random;

import java.util.Scanner;
public class Usuario {
    private int id;
    private String nombre;
    private int edad;
    private ArrayList<String> rentaLibros;

    public Usuario() {

    }
    public Usuario(String nombre, int edad, int id) {
        Random random = new Random();
        this.id = id;
        this.nombre = nombre;
        this.edad = edad;
        this.rentaLibros = new ArrayList<>();
    }
    public String getNombre() {
        return nombre;
    }
    public void setNombre() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Ingresa tu nombre: ");
        this.nombre = scanner.nextLine();
    }
    public int getEdad() {
        return edad;
    }
    public void setEdad() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Ingresa tu edad: ");
        this.edad = scanner.nextInt();
    }

    public int getId() {
        return id;
    }

    public void setId() {
        Random random = new Random();
        this.id = random.nextInt(1000) + 1;
    }
}

File: Tarea8Python/Usuario.py
import random

class Usuario:
    def __init__(self, nombre=None, edad=None, id=None):
        self.id = id
        self.nombre = nombre
        self.edad = edad

    def get_nombre(self):
        return self.nombre

    def set_nombre(self):
        self.nombre = input("Ingresa tu nombre: ")

    def get_edad(self):
        return self.edad

    def set_edad(self):
        self.edad = int(input("Ingresa tu edad: "))

    def get_id(self):
        return self.id

    def set_id(self):
        self.id = random.randint(1, 1000)
